.env.example

```
@@ -2,96 +2,203 @@
+# admin:test
+# Note, password protection does not currently function. Only enable the dashboard for testing.
+TRAEFIK_DASH_ENABLE=false
+TRAEFIK_ACCESS_HASH='admin:$5$CF7jWr0NxTEcg4/v$GNfwPDvivpY7cueLHIYalEKjtdeIKe/WrP8iKp9CAc9'
+CLOUDFLARE_IPS="173.245.48.0/20, 103.21.244.0/22, 103.22.200.0/22, 103.31.4.0/22, 141.101.64.0/18, 108.162.192.0/18, 190.93.240.0/20, 188.114.96.0/20, 197.234.240.0/22, 198.41.128.0/17, 162.158.0.0/15, 104.16.0.0/13, 104.24.0.0/14, 172.64.0.0/13, 131.0.72.0/22, 2400:cb00::/32, 2606:4700::/32, 2803:f800::/32, 2405:b500::/32, 2405:8100::/32, 2a06:98c0::/29, 2c0f:f248::/32"
+#No limits:
+#TRAEFIK_TRUSTED_IPS="0.0.0.0/0, 0::/0"
+#Only allow traffic from CloudFlare:
+TRAEFIK_TRUSTED_IPS=$CLOUDFLARE_IPS
 DB_NAME=pece
 DB_USER=pece
 DB_PASSWORD=pece
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USER key

(UnorderedKey)